apps/dashboard/components/logs/queries/list-group.tsx

```
@@ -0,0 +1,216 @@
+          <Tooltip open={toolTipOpen}>
+            <TooltipTrigger>
+              <div
+                className={cn(
+                  "flex h-7 w-6 ml-[1px]  justify-center items-center text-accent-9 rounded-md",
+                  isSaved ? "text-info-9 hover:bg-info-3" : "hover:bg-gray-3 hover:text-accent-12",
+                  `tabIndex-${0}`,
+                )}
+                role="button"
+                onClick={handleBookmarkChanged}
+                onKeyUp={(e) => e.key === "Enter" && console.log("Saved", index)}
+              >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)